model/src/main/java/org/cbioportal/model/summary/MutationSummary.java

```diff
@@ -0,0 +1,259 @@
+package org.cbioportal.model.summary;
+
+import java.io.Serializable;
+
+public abstract class MutationSummary implements Serializable {
```

Comment:
Indeed, there's already the ORM impedance mismatch problem to deal with for the persistence model. The actual domain model needs to be completely decoupled from the shortcomings of the persistence layer (and I would recommend a Repository layer in between, so the backend can be abstracted away), this ought to help turning the very fat services like `ApiService` into the thinest layer possible, for the very limited logic that truly does not belong in the objects themselves (if any).